packages/nx/src/tasks-runner/forked-process-task-runner.ts

```diff
@@ -203,6 +203,7 @@ export class ForkedProcessTaskRunner {
       execArgv: process.execArgv,
       jsEnv: env,
       quiet: !streamOutput,
+      commandLabel: `nx run ${task.id}`,
```

Comment:
This doesn't seem like the right label? We want the underlying tool right? We already have the task id printed on the terminal output pane, we want to know what that task did?

Reply:
This `fork` call is for runs that use executors, not run-commands